README.md

```diff
@@ -146,6 +146,140 @@ Module blocks could be transpiled to either data URLs, or to a module in a separ
 
 This proposal only allows anonymous module blocks. There are other proposals for named module *bundles* (with URLs corresponding to the specifier of each JS module), including "[JS Module Bundles]" proposal, and [Web Bundles](https://www.ietf.org/id/draft-yasskin-wpack-bundled-exchanges-03.html). Note that there are significant privacy issues to solve with bundling to permit ad blockers; see [concerns from Brave](https://brave.com/webbundles-harmful-to-content-blocking-security-tools-and-the-open-web/).
 
+## FAQs
+
+### Should we really allow creation of workers using module blocks?
+
+In my opinion: Yes. The requirement that workers are in a separate file is one of the most prominent pieces of feedback about why workers are hard to adopt. That’s why so many resort to Blob URLs or Data URLs, bringing along all kinds of difficulties, especially relating to paths and CSP. The risk here is that people start spawning a lot workers without regards to their cost, but I think the benefits of lowering the barrier to workers as an important performance primitive outweigh the risks.
```

Comment:
This would be a good place to link to the issue. Also, maybe put it further down in the list, as it's not really a *frequently* asked question (whereas the next four questions are actually frequent).